client/network/src/lib.rs

```diff
@@ -230,52 +225,39 @@ impl TryFrom<&'_ CollationSecondedSignal> for BlockAnnounceData {
 /// chain. If it is at the tip, it is required to provide a justification or otherwise we reject
 /// it. However, if the announcement is for a block below the tip the announcement is accepted
 /// as it probably comes from a node that is currently syncing the chain.
-pub struct BlockAnnounceValidator<Block, R, B, BCE> {
+pub struct BlockAnnounceValidator<Block, RCInterface> {
```

Comment:
Sometimes you call it `R`, sometimes `RCInterface`. Maybe a little bit consistency would be nice :P